index.html

```diff
@@ -6,15 +6,27 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
+    <script src="menu.js" defer></script>
   </head>
   <body>
     <div class="background--c"></div>
     <header class="navbar">
       <div class="mylogo">
         <a href="#">Elhama Tokhi</a>
       </div>
+      <div class="hamburger">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+      </div>
+      <div class="navMobile">
+        <ul>
+          <li class="link"><a href="#" >Contact</a></li>
+          <li class="link"><a href="#" >About</a></li>
+          <li class="link"><a href="#" >Portfolio</a></li>
```

Comment:
- [ ] Kindly add the name of the id for each section instead of #, so when you click on any option it goes to the specific section.

like : `<a href="#contact" >Contact</a>`